pkg/build/gobuild.go

```diff
@@ -82,26 +85,54 @@ func NewGo(options ...Option) (Interface, error) {
 	return gbo.Open()
 }
 
+// goList is a wrapper for the `go list` command to retrieve information about a package.
+func (g *gobuild) goList(importPath, projection string) (string, error) {
```

Comment:
nit: there's a lot of extra newlines in here, I'd drop most of them (I usually like to leave a newline after an error handling clause as a kind of visual "checkpoint")

Reply:
Thanks, this was one of the first codes in go I have written and had a lot of bad habit from other languages :)